packages/graphql/src/schema-model/entity/ConcreteEntity.ts

```diff
@@ -40,11 +55,18 @@ export class ConcreteEntity implements Entity {
 
     private addAttribute(attribute: Attribute): void {
         if (this.attributes.has(attribute.name)) {
-            throw new Error(`Attribute ${attribute.name} already exists in ${this.name}`);
+            throw new Neo4jGraphQLSchemaValidationError(`Attribute ${attribute.name} already exists in ${this.name}`);
```

Comment:
Potentially if we define the ConcreteEntity to be a mere representation of the GraphQL type then this validation could be anticipated by the GraphQL validation phase. Just a comment, no need to be changed!

Reply:
I guess this file might also throw errors related to our parsing code, rather than just the GraphQL types. Also the logical operators combination.. What do you think?
I wouldn't remove the throw, but maybe go back to the generic Error?

Reply:
I believe that we can keep it like this for now and then with the validation lifecycle discussion, we can discuss when and how this throw should happen. But, IMO, it is such a minor thing that at this moment we can ignore it!